spring-cloud-generator/scripts/setup-googleapis-rules.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+cd googleapis
+
+# add local_repository rule with name "spring_cloud_generator"
+buildozer 'new local_repository spring_cloud_generator before gapic_generator_java' WORKSPACE:__pkg__
+# point path to local repo
+buildozer 'set path "../../spring-cloud-generator"' WORKSPACE:spring_cloud_generator
+
+# delete existing maven_install rules
+buildozer 'delete' WORKSPACE:%maven_install
+
+# In googleapis/WORKSPACE, find maven_install() rule for gapic-generator-java jar, and remove
+# perl -0777 -pi -e "s{maven_install\(\n    artifacts = \[(.*?)_gapic_generator_java_version(.*?)\](.*?)\)}{}s" WORKSPACE
+
+# add custom maven_install rules
+perl -pi -e "s{(^_gapic_generator_java_version[^\n]*)}{\$1\n$(cat ../scripts/resources/googleapis_modification_string.txt)}" WORKSPACE
```

Comment:
Modifies googleapis/WORKSPACE file to use similar approach as gapic libraries generation: build spring-cloud-generator local jar, then pass this into bazel plugin to generate the spring modules.